libs/api/domains/cms/src/lib/dto/getLifeEvents.input.ts

```diff
@@ -1,8 +1,13 @@
 import { Field, InputType } from '@nestjs/graphql'
-import { IsString } from 'class-validator'
+import { IsString, IsOptional } from 'class-validator'
 
 @InputType()
 export class GetLifeEventsInput {
+  @Field({ nullable: true })
```

Comment:
It's better IMO to not allow null unless you have a specific reason to.
Is there any reason here to allow null?

Reply:
The reason is that I'm using the same type as we do to fetch **all** LifeEvents. Perhaps we should keep them separate? As in getAllLifeEvents.input.ts and getLifeEventsInCategory.input.ts ?